scripts/slave_tab.gd

```diff
@@ -1,4 +1,6 @@
 
+#var showfullbody = true
```

Comment:
This file is reverted and slave namecolors are removed.

Reply:
fixed for next commit